src/config/main.ejs

```diff
@@ -188,6 +188,7 @@ module.exports = {
       // https://www.browserstack.com/automate/capabilities?tag=selenium-4
       desiredCapabilities: {
         'bstack:options': {
+          "buildName": "nightwatch-test-build",
```

Comment:
This will solve the issue for the new users, but for old users who aren't using `bstack:options` would still see `undefined` in the URL.

Related issue: https://github.com/nightwatchjs/nightwatch/issues/3149
Related Fix which was merged: https://github.com/nightwatchjs/nightwatch/pull/3148